cmd/cli/kubectl-kyverno/utils/common/common.go

```diff
@@ -505,16 +503,6 @@ OuterLoop:
 	}
 
 	if policyHasGenerate {
-		policyContext := &engine.PolicyContext{
-			NewResource:      *c.Resource,
-			Policy:           c.Policy,
-			ExcludeGroupRole: []string{},
-			ExcludeResourceFunc: func(s1, s2, s3 string) bool {
-				return false
```

Comment:
What are these two vars? Doesn't seem to be needed.

Reply:
What do you mean ?
This code was removed.

Reply:
Just questioning why they were added before, nothing to change :)